docs/components/webapp.rst

```diff
@@ -43,6 +43,53 @@ Then you can start and stop the webapp, adjust files, and debug.
 FIXME(willkg): Review everything after this line.
 
 
+Setting up authentication and a superuser
+=========================================
+
+Creating a superuser
+--------------------
+
+If you want to do anything in the webapp admin, you'll need to create a
+superuser.
+
+Run this::
+
+  $ docker-compose run webapp python manage.py createsuperuser
```

Comment:
We have our own command and it's called [`makesuperuser`](https://github.com/mozilla-services/socorro/blob/master/webapp-django/crashstats/authentication/management/commands/makesuperuser.py)
It doesn't bother with username and password and works for both existing and new users.